src/test/extensions/provisioningHelper.spec.ts

```diff
@@ -0,0 +1,210 @@
+import * as nock from "nock";
+import * as api from "../../api";
```

Comment:
Nit: Split up internal and external imports.

Reply:
Done.